doc/README_docker_gnu.md

```diff
@@ -151,11 +150,11 @@ The files:
    set the LAYOUT_X, LAYOUT_Y, RUN_CMD_UTILS, and RUN_CMD_POST
    variables.
 
-3. LOW MEMORY MACHINES - The workflow takes about 16 GB of RAM, on top
-   of the memory your OS and other applications use. If you don't have
-   significantly more than 16 GB of RAM, then use the 4 core config,
-   but reduce the utilities to one MPI rank. Do that by putting this
-   at the end of config.sh:
+3. LOW MEMORY MACHINES - The workflow used more than 16 GB of RAM, on
```

Comment:
Change to "*uses* more than 16 GB of RAM".

A side note: my machine only has 16 GB of RAM, but I was able to run the app successfully with the RAM+swap space allocated to Docker (20 GB total). Not sure if that's worth a mention though.